packages/manager/modules/sms/src/sms/components/service-status/service-status.html

```diff
@@ -0,0 +1,62 @@
+<oui-tile
+    data-heading="{{ 'sms_component_serviceStatus_title' | translate }}"
+    data-loading="$ctrl.loading"
+>
+    <oui-tile-definition
+        data-term="{{ 'sms_component_serviceStatus_label_status' | translate }}"
+    >
+        <oui-tile-description>
+            <span
+                class="oui-badge"
+                data-ng-class="{
+                    'oui-badge_error': $ctrl.status === 'ERROR' || $ctrl.status === 'disable',
+                    'oui-badge_success': $ctrl.status === 'OK' || $ctrl.status === 'enable',
+                    'oui-badge_info': $ctrl.status === 'INSTALLING' || $ctrl.status === 'UPDATING_IPS',
```

Comment:
May be having status into const file is more scalable.

Reply:
It's fixed. I fixed the class in constant.